app/script/localization/webapp.js

```diff
@@ -45,7 +45,8 @@ z.string.auth_account_sign_in_phone = 'Phone';
 
 // Authentication: BLOCKED section
 z.string.auth_blocked_cookies = 'Enable cookies to log in to Wire.';
-z.string.auth_blocked_database = 'Wire needs access to local storage to display your messages. Local storage is not available in private mode.';
+z.string.auth_blocked_database =
+  'Wire needs access to local storage to display your messages. Local storage is not available in private mode.';
```

Comment:
Please no prettier changes to the strings file 🙈

Reply:
https://github.com/wireapp/wire-webapp/blob/dev/.prettierrc.yaml maybe we can add another override for these files with huge printWidth